catroid/src/org/catrobat/catroid/content/bricks/DroneMoveBrick.java

```diff
@@ -45,6 +45,7 @@
 import java.util.List;
 
 public abstract class DroneMoveBrick extends FormulaBrick {
+	private static final String TAG = ProjectManager.class.getSimpleName();
```

Comment:
You should use the TAGs corresponding to the class rather than the ProjectManager's TAG.